src/UserALEWebExtension/messageTypes.ts

```diff
@@ -20,3 +20,4 @@ const prefix = "USERALE_";
 export const CONFIG_CHANGE = prefix + "CONFIG_CHANGE";
 export const ADD_LOG = prefix + "ADD_LOG";
 export const HTTP_SESSION = prefix + "HTTP_SESSION";
+export const ISSUE_REPORT = prefix + "ISSUE_REPORT";
```

Comment:
I recommend we use an Enum here. That's essentially how you're using the import in other locations anyways.

Reply:
Agreed, will change.